### DISCUSSION TITLE
Jamulus' display of Qt versions

### DISCUSSION THREAD

**Discussion by pljones** [jamulussoftware/jamulus#3356] [General]
From https://github.com/jamulussoftware/jamulus/pull/3325#pullrequestreview-2246104367

On running with "Jamulus --version", running headless or in "Help->About Jamulus..."->"Libraries", Jamulus displays the versions of the Qt libraries it's using:

e.g. my own brew Linux build:
```
 *** Qt cross-platform application framework 6.7.2 (build), 6.7.2 (runtime)
```
and the Github Windows tagged beta:
> Qt cross-platform application framework 6.6.3 (build), 6.6.3 (runtime), https://www.qt.io/

Translators have the words
> (build)
> (runtime)

to translate with no context about their intent, which can result in potentially unintentional translation.

My understanding is that the Qt version
- labelled "build" was the version of Qt installed on the platform where the Jamulus executable was compiled and linked -- in common developer jargon, "built".
- labelled "runtime" is the version of Qt being used by Jamulus on the platform where it is executing -- in common developer jargon, "runtime".

So the current terms are not _wrong_ - but do not carry the above context.

Proposed alternatives for the English which might enable more contextually helpful translation are:
- "build" becomes "compile"
- "runtime" becomes "execution"

Alternatively, the Weblate files need additional context provided to explain the intent so that translators can choose a more appropriate term.

CC: @jamulussoftware/maindevelopers @jamulussoftware/translators

**Comment by ann0see** [comment]
There was also a discussion about writing a short sentence. I think something like Built with Qt x, running with Qt y would be ok.

**Reply by jujudusud** [reply]
For information:
`$ pipewire --version`
`pipewire
Compiled with libpipewire 1.2.3
Linked with libpipewire 1.2.3`
